packages/web-components/readme.md

```diff
@@ -16,22 +21,19 @@ Stencil components are just Web Components, so they work in any major framework
 
 ## Getting Started
 
-To start building a new web component using Stencil, clone this repo to a new directory:
-
-```bash
-git clone https://github.com/ionic-team/stencil-component-starter.git my-component
-cd my-component
-git remote rm origin
-```
+To start building a new web component using Stencil:
 
-and run:
+run:
 
 ```bash
 npm install
-npm start
+npm serve
```

Comment:
We don't really use the `serve` command much, do we? I think we usually use `watch:stencil` in conjunction with `yarn storybook`, but maybe other people do it differently.

Also, should include instructions to cd into the `packages/web-components` directory before running the watch command.

Reply:
And likewise cd into the `packages/storybook` directory before running the storybook command.

Reply:
> We don't really use the serve command much, do we? I think we usually use watch:stencil in conjunction with yarn storybook

@Andrew565 I was considering this too but I think these instructions make sense when using `web-components` as a [standalone package](https://www.npmjs.com/package/@department-of-veterans-affairs/web-components) (I haven't tested it outside of the monorepo structure yet though).

Reply:
I can add a section for developing in storybook. I've used the npm serve command before though and it works fine. Faster than dealing with storybook and one less thing to juggle.

Reply:
> I can add a section for developing in storybook. 

I sort of think it would be redundant with what the monorepo/core readme already has: https://github.com/department-of-veterans-affairs/component-library?tab=readme-ov-file#running-build-via-storybook

Reply:
Fair. What about something like: 

"There are two ways this project can be run. For running it with storybook follow [these steps](https://github.com/department-of-veterans-affairs/component-library?tab=readme-ov-file#running-build-via-storybook). For just running the stencil dev server, follow the steps below."